### PR TITLE
Update portuguese stations

### PR DESCRIPTION
**Pull request by paulRbr**

### CODE REVIEW

File: stations.csv [hunk 1]
```diff
@@ -7256,43 +7256,43 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;is_city;country;
 8672;Schiphol Airport;schiphol-airport;8400561;;52.3087870000;4.7614870000;;f;NL;f;Europe/Amsterdam;t;NLSPH;;t;;f;8400561;t;;f;;f;;f;;f;;f;;f;f;;Amsterdam;Amsterdam;Amsterdam;Amsterdam;Amsterdam;Amsterdam;Amsterdam;Amsterdam;Amszterdam;アムステルダム;암스테르담;Amsterdam;Amesterdão;Амстердам;Amsterdam;Amsterdam;阿姆斯特丹
 8673;Utrecht Centraal;utrecht-centraal;8400621;;52.0875800000;5.1111670000;;f;NL;f;Europe/Amsterdam;t;NLUTC;;f;;f;8400621;t;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;ユトレヒト;;;;Утрехт;;;
 8674;Pinhel;pinhel;9448595;;;;;f;PT;f;Europe/Lisbon;f;PTAAA;;f;;f;;f;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;
-8675;Vila Franca das Naves;vila-franca-das-naves;9448546;;;;;f;PT;f;Europe/Lisbon;f;PTAAB;;f;;f;;f;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;
-8676;Mangualde;mangualde;9448009;;;;;f;PT;f;Europe/Lisbon;f;PTAAC;;f;;f;;f;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;
+8675;Vila Franca das Naves;vila-franca-das-naves;9448546;;;;;f;PT;f;Europe/Lisbon;f;PTAAB;;f;;f;;f;;f;;f;;f;;f;;f;94546;t;f;;;;;;;;;;;;;;;;;;
```
Comment (mgalibert) [inline stations.csv]: Pas suggestable c'est normal ?

Comment (paulRbr) [inline stations.csv]: Je la rajoute

File: stations.csv [hunk 2]
```diff
@@ -7256,43 +7256,43 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;is_city;country;
...
 8700;Lagos;lagos;9490464;;;;;f;PT;f;Europe/Lisbon;f;PTABE;;f;;f;;f;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;
 8701;Lisboa Terreiro do Paço;lisboa-terreiro-do-paco;9498004;;;;;f;PT;f;Europe/Lisbon;f;PTABF;;f;;f;;f;;f;;f;;f;;f;;f;;f;f;;Lissabon;Lisbon;;Lisbonne;Lisbona;Lissabon;Lisabon;Lissabon;Lisszabon;;;Lizbona;;Лиссабон;Lissabon;;
 8702;Portimão;portimao;9490290;;;;;f;PT;f;Europe/Lisbon;f;PTABG;;f;;f;;f;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;
-8703;Vila Real de San António Guad;vila-real-de-san-antonio-guad;9478964;;;;;f;PT;f;Europe/Lisbon;f;PTABH;;f;;f;;f;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;
+8703;Vila Real de San António Guad;vila-real-de-san-antonio-guad;9478964;;37.19979;-7.421566;;f;PT;f;Europe/Lisbon;f;PTABH;;f;;f;9400632;f;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;
 8704;Beja;beja;9430011;;;;;f;PT;f;Europe/Lisbon;f;PTABI;;f;;f;;f;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;
-8705;Valença;valenca;9407005;;;;;f;PT;f;Europe/Lisbon;f;PTABJ;;f;;f;;f;;f;;f;;f;;f;;f;;f;f;;;Valenza;;;;;;;;;;;;Валенса;;;
+8705;Valença;valenca;9407005;;42.024705;-8.639280;;f;PT;f;Europe/Lisbon;t;PTABJ;;f;;f;9400648;f;;f;;f;;f;;f;;f;22402;t;f;;;Valenza;Valenza Do Minho;;;;;;;;;;;Валенса;;;
```
Comment (mgalibert) [inline stations.csv]: `Valenza do Minho`

File: stations.csv (hunk 1 above)
Comment (paulRbr) [inline stations.csv]: Updated

File: stations.csv (hunk 2 above)
Comment (paulRbr) [inline stations.csv]: Updated

**Pull request merged**